specs/components/recipeCard/altRecipeCard.html

```diff
@@ -11,21 +11,26 @@
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
   <!-- Local -->
   <link rel="stylesheet" href="./altRecipeCard.css">
+  <link rel="stylesheet" href="./iconRecipeCard.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
+
 <!-- Scripts -->
 <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+<script src="https://kit.fontawesome.com/0e68289b8e.js" crossorigin="anonymous"></script>
```

Comment:
It is recommended that the bootstrap script be placed at the end of the body, prior to the closing body tag. I'll look into whether this makes any difference for both bootstrap and font awesome.